### PR TITLE
Selectively load Sass

### PR DESCRIPTION
**Pull request by injms**
## What

Only importing the Sass that is needed by the app.

I've gone through and checked each question, plus the error state, the ineligible pages, plus the privacy notice and all look identical to me.

## Why

It'll reduce the amount of CSS users need to load, speeding up the service.

Before: 349.36KB of CSS
After: 116.87KB of CSS
With minification: 96.49KB of CSS

## Visual differences

None.

### CODE REVIEW

**Comment by NickColley** [comment]
Worth testing each page and double checking but on paper this is a great change.

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ gem "log_formatter", "~> 0.8.2"
 gem "puma", "~> 4.3"
 gem "sentry-raven", "~> 3.0"
 gem "uglifier", "~> 4.2"
+gem "sassc-rails"
```
Comment (NickColley) [inline Gemfile]: think this'll need to be pinned for it to work

Comment (injms) [reply Gemfile]: argh, of course

Comment (injms) [reply Gemfile]: Now changed to `sass-rails` and pinned to less than 6.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1,4 +1,31 @@
-@import 'govuk_publishing_components/all_components';
+// @import 'govuk_publishing_components/all_components';
+
+@import 'govuk_publishing_components/govuk_frontend_support';
```
Comment (NickColley) [inline app/assets/stylesheets/application.scss:3]: inset text is missing

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -1,4 +1,31 @@
-@import 'govuk_publishing_components/all_components';
+// @import 'govuk_publishing_components/all_components';
+
```
Comment (NickColley) [inline app/assets/stylesheets/application.scss:2]: can you check you've got the styles needed for the confirmation page please?

**Comment by NickColley** [comment]
Noticed two components required.

Would be good to get another person to visit every possible page in the journey and check nothing is missing before merging.

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (injms) [reply app/assets/stylesheets/application.scss:3]: thanks! added

**Comment by injms** [comment]
> Noticed two components required.
> 
> Would be good to get another person to visit every possible page in the journey and check nothing is missing before merging.

I've added inset text - what was the other component?

**Comment by NickColley** [comment]
Panel for the confirmation

**Comment by NickColley** [comment]
@injms could you rebase your branch and double check all pages please :pray:

**Comment by injms** [comment]
> @injms could you rebase your branch and double check all pages please 🙏

rebased and checked x2, including the final page and the ineligible pages.

**Review by Nooshu — APPROVED** [review]
Approve to be merged. I will check on staging once merged.

**Review by NickColley — APPROVED** [review]

**Pull request merged**